static/templates/message_info_popover_content.handlebars

```diff
@@ -27,4 +27,9 @@
       <i class="icon-vector-bullhorn"></i> {{#tr this}}Narrow to messages sent by __message.sender_full_name__{{/tr}}
     </a>
   </li>
+  <li>
+    <a class="mention_user">
+      {{#tr this}}@ Mention __message.sender_full_name__ in next message{{/tr}}
```

Comment:
This should probably read "Compose a reply @-mentioning __message.sender_full_name__" for greater clarity?